/**:                                               #GNSS cycle 5Hz, IMU cycle 50Hz.
  ros__parameters:
# Using Topic Name
    twist_topic: "can_twist"
    imu_topic: "imu/data_raw"
    rtklib_nav_topic: "rtklib_nav"
    navsatfix_topic: "navsat/fix"
    nmea_sentence_topic: "nmea_sentence"
